docs/content/2.deploy/20.providers/cloudflare.md

```diff
@@ -73,183 +148,51 @@ Publish:
 npx wrangler deploy
 ```
 
-### Deploy within CI/CD using GitHub Actions
-
-Create a token according to [the wrangler action docs](https://github.com/marketplace/actions/deploy-to-cloudflare-workers-with-wrangler#authentication) and set `CLOUDFLARE_API_TOKEN` in your repository config on GitHub.
+## Cloudflare Service Workers
 
-Create `.github/workflows/cloudflare.yml`:
-
-```yaml
```

Comment:
Here I thought that showing how to use the wrangler actions isn't really a Nitro concern so I removed this and just changed this section referring people to the official wrangler actions docs, hopefully this is fine, if it isn't I can revert this